tools/dotcms-cli/cli/src/main/java/com/dotcms/api/client/pull/PullHandler.java

```diff
@@ -14,20 +18,13 @@ public interface PullHandler<T> {
     String title();
 
     /**
-     * Returns a display name of a given T element. Used for logging purposes.
-     */
-    String displayName(T content);
-
-    /**
-     * Returns the file name for a given T elements used to save the content to a file.
-     *
-     * @param content the content to be saved to a file.
+     * Returns a header for the T elements being pulled. Used for console user feedback.
      */
-    String fileName(T content);
+    String startPullingHeader(List<T> contents);
 
     /**
      * Returns a short format of a given T element. Used for console user feedback.
      */
-    String shortFormat(T content);
+    String shortFormat(T content, PullOptions pullOptions, Map<String, Object> customOptions);
```

Comment:
might be customOptions could be a property into the pull options to avoid a lot of arguments

Reply:
Good idea, and done, now `customOptions` is a property of the `PullOptions`.